### ISSUE TITLE
Publish new `av-data` release to crates.io to remove syn v1 dependency

### ISSUE THREAD

**Issue by FreezyLemon** [rust-av/rust-av#187]
After #186, av-data no longer depends on syn v1. It would be nice to have a crates.io release with the change.

av-codec depends on av-data, so it should be updated after the release.

**Comment by FreezyLemon** [comment]
I've published data-v0.4.2 which includes the change from #186.
